src/metabase/models/card.clj

```diff
@@ -850,75 +850,66 @@
                                     [(when (:table_id m) #{(serdes/table->path (:table_id m))})
                                      (when (:id m)       #{(serdes/field->path (:id m))})])))))
 
-(defmethod serdes/extract-one "Card"
-  [_model-name _opts card]
-  ;; Cards have :table_id, :database_id, :collection_id, :creator_id that need conversion.
-  ;; :table_id and :database_id are extracted as just :table_id [database_name schema table_name].
-  ;; :collection_id is extracted as its entity_id or identity-hash.
-  ;; :creator_id as the user's email.
-  (-> (serdes/extract-one-basics "Card" card)
-      (update :database_id            serdes/*export-fk-keyed* 'Database :name)
-      (update :table_id               serdes/*export-table-fk*)
-      (update :collection_id          serdes/*export-fk* 'Collection)
-      (update :creator_id             serdes/*export-user*)
-      (update :made_public_by_id      serdes/*export-user*)
-      (update :dataset_query          serdes/export-mbql)
-      (update :parameters             serdes/export-parameters)
-      (update :parameter_mappings     serdes/export-parameter-mappings)
-      (update :visualization_settings serdes/export-visualization-settings)
-      (update :result_metadata        export-result-metadata)
-      (dissoc :cache_invalidated_at :view_count :last_used_at :initially_published_at
-              :dataset_query_metrics_v2_migration_backup)))
-
-(defmethod serdes/load-xform "Card"
-  [card]
-  (-> card
-      serdes/load-xform-basics
-      (update :database_id            serdes/*import-fk-keyed* 'Database :name)
-      (update :table_id               serdes/*import-table-fk*)
-      (update :creator_id             serdes/*import-user*)
-      (update :made_public_by_id      serdes/*import-user*)
-      (update :collection_id          serdes/*import-fk* 'Collection)
-      (update :dataset_query          serdes/import-mbql)
-      (update :parameters             serdes/import-parameters)
-      (update :parameter_mappings     serdes/import-parameter-mappings)
-      (update :visualization_settings serdes/import-visualization-settings)
-      (update :result_metadata        import-result-metadata)))
+(defmethod serdes/make-spec "Card"
+  [_model-name]
+  {:copy [:name :description :archived :collection_position :enable_embedding :type :query_type
+          :display :archived_directly :entity_id :collection_preview :metabase_version :public_uuid
+          :created_at :embedding_params]
+   :skip [:id :updated_at :cache_invalidated_at :view_count :last_used_at :initially_published_at
+          :dataset_query_metrics_v2_migration_backup :cache_ttl]
+   :transform
+   {:database_id            {:ser #(serdes/*export-fk-keyed* % 'Database :name)
+                             :des #(serdes/*import-fk-keyed* % 'Database :name)}
```

Comment:
Given how repetitive this stuff is, I kinda want to clean it up with some convention-over-configuration or metadata based trickery. You would hate that, right? 😄

Reply:
haha, I've been thinking about one var in there, like `*fk-keyed*` or something like that, but I don't want to abstract too early. My idea is to bring declarative syntax first, and then clean it up later on.

Reply:
Having one var sounds great to me. The end result would be similar to `t2/deftransforms` with `mi/transform-keyword` and `mi/transform-json`, e.g. 
https://github.com/metabase/metabase/blob/52a819b312cb0b05fd0f9318e4016ae7bd7c079c/src/metabase/models/audit_log.clj#L29-L31